### PR TITLE
ceph/doc: Add guidance for cidr network troubleshooting

### PR DESCRIPTION
**Pull request by zdover23**
## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard backend`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

**Comment by sebastian-philipp** [comment]
maybe something like this helps?

```
git checkout master
git fetch upstream master
git reset --hard upstream/master
git checkout -b wip-doc-cidr-network-troubleshooting3
git cherry-pick 6abec25e11499d244f05fd6e208495be77e1685d
git checkout  wip-doc-cidr-network-troubleshooting2
git reset --hard wip-doc-cidr-network-troubleshooting3
git push -f
```
works for you. please be aware that the `reset` commands are a bit forceful.

**Comment by sebastian-philipp** [comment]
jenkins render docs

File: doc/cephadm/troubleshooting.rst [hunk 1]
```diff
@@ -102,3 +102,16 @@ Manually running containers
...
+
+If you see this error::
+
+   ERROR: Failed to infer CIDR network for mon ip ***; pass --skip-mon-network to configure it later
+
+This means that you must run a command of this form::
+
+  ceph config set mon public_network <mon_network>
+
+For more detail on operations of this kind, see https://docs.ceph.com/docs/master/cephadm/install/?highlight=public_network#deploy-additional-monitors-optional
```
Comment (sebastian-philipp) [inline doc/cephadm/troubleshooting.rst:117]: can we use a proper REST  `:ref:...` link here? See the Sphinx part of https://stackoverflow.com/a/19543591/3185053

Comment (zdover23) [reply doc/cephadm/troubleshooting.rst:117]: This change is in.

Comment (sebastian-philipp) [reply doc/cephadm/troubleshooting.rst:117]: no. did you push this?

Comment (zdover23) [reply doc/cephadm/troubleshooting.rst]: At long last, I did. Every single time, this kind of operation is Darmok and Jilad at Tenagra to me.

**Comment by sebastian-philipp** [comment]
jenkins retest this please

**Comment by sebastian-philipp** [comment]
> At long last, I did. Every single time, this kind of operation is Darmok and Jilad at Tenagra to me.

Very sorry to say that, but you'll need another round to rebase this.

**Comment by sebastian-philipp** [comment]
I think you'll have do do another `git rebase -i master` round and squash all of those commits.

**Comment by zdover23** [comment]
I get a clean build from 36219fbb03f1c10ff29343d597af2b16bce251c4 ,
so I don't know what github is on about when it says that the Docs: build check fails.

**Comment by sebastian-philipp** [comment]
jenkins retest this please

**Comment by tchaikov** [comment]
@zdover23 i rebased your change onto master and removed unrelated changes to submodules.

**Pull request merged**